docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,305 @@
-Now you can watch your pytorch-lightning model training in Neptune!
+Step 5 - Create NeptuneLogger
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``NeptuneLogger`` is an object that integrates Neptune with PyTorch Lightning allowing you to track your experiments in Neptune. It's a part of the lightning library.
 
-Check out this |example experiment|.
-
-.. image:: ../_static/images/integrations/pytorch_lightning_monitoring.gif
-   :target: ../_static/images/integrations/pytorch_lightning_monitoring.gif
-   :alt: PyTorchLightning logging in neptune
-
-Full PyTorch Lightning monitor script
--------------------------------------
-Simply copy and paste it to ``pytorch_lightning_example.py`` and run.
-You can change your credentials in ``NeptuneLogger`` or run some tests as an anonymous user:
+In this minimalist example we use public user `"neptuner"`, who has public token: "ANONYMOUS".
 
 .. code-block:: python3
 
+    from pytorch_lightning.loggers.neptune import NeptuneLogger
+
     neptune_logger = NeptuneLogger(
         api_key="ANONYMOUS",
         project_name="shared/pytorch-lightning-integration",
-        ...
-        )
+        params=PARAMS)
+
+.. tip::
+
+    Make sure to use your API token in your projects. Read more about how to |token|.
+
+Step 6 - Fit model to the data - Neptune tracks it automatically
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+At this point you are ready to fit the model to the data. Simply pass ``neptune_logger`` to the ``Trainer`` and run ``fit()`` loop. Neptune will collect metrics and show them in UI.
+
+Notice, that ``max_epochs`` is from the ``PARAMS`` dictionary. All these params are logged to Neptune.
 
 .. code-block:: python3
 
-    import os
+    trainer = pl.Trainer(max_epochs=PARAMS['max_epochs'],
+                         logger=neptune_logger)
+    model = LitModel()
 
-    import torch
-    from torch.nn import functional as F
-    from torch.utils.data import DataLoader
-    from torchvision.datasets import MNIST
-    from torchvision import transforms
+    trainer.fit(model, train_loader)
 
-    import pytorch_lightning as pl
+Results
+^^^^^^^
+You just learned how to start logging PyTorch Lightning experiments to Neptune, by using Neptune logger which is part of the lightning library.
 
-    MAX_EPOCHS=20
-    LR=0.01
-    BATCHSIZE=32
-    CHECKPOINTS_DIR = 'my_models/checkpoints/7'
+Above training is logged to Neptune in near real-time. You can go and explore the results. In particular check:
 
-    class CoolSystem(pl.LightningModule):
+#. |metrics|,
+#. |params|,
+#. |hardware|,
+#. |metadata|.
 
-        def __init__(self):
-            super(CoolSystem, self).__init__()
-            # not the best model...
-            self.l1 = torch.nn.Linear(28 * 28, 10)
+.. image:: ../_static/images/integrations/pytorchlightning_neptuneml.png
+   :target: ../_static/images/integrations/pytorchlightning_neptuneml.png
+   :alt: PyTorchLightning neptune.ai integration
```

Comment:
some link to experiment shown in image would be good I think below (I know there are some up but you know how it is... :) )

Reply:
I'd also add Colab button somewhere here -> in case someone feels actionable enough to try it.

Reply:
added colab and link.